src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const littleDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return dayPrice * days;
+  } else if (days >= 3 && days <= 6) {
```

Comment:
`days >= 3` check is useless here, we already passed it in line 35.

```suggestion
  if (days < 3) {
    return dayPrice * days;
  } else if (days <= 6) {
```